public/locales/en/modules/rss.json

```diff
@@ -0,0 +1,20 @@
+{
+  "descriptor": {
+    "name": "RSS Widget",
+    "description": "Grabs the items from a RSS feed and displays them. Commonly used for online news",
+    "settings": {
+      "title": "Settings for RSS widget",
+      "rssFeedUrl": {
+        "label": "RSS feed url"
+      }
+    }
+  },
+  "card": {
+    "errors": {
```

Comment:
There is no need for so much nesting if there is no alternatives. It just adds complexity to the trasnlations. (Could just be `errorTitle` and `errorDescription`)

Reply:
In my opinion nesting makes total sense here

Reply:
I disagree. It makes sense to read and is easier to adjust, if more errors are added.